torch/csrc/jit/tensorexpr/schedule.cpp

```diff
@@ -733,10 +731,15 @@ SplitAxisWithTail::SplitAxisWithTail(
   // TODO: support factor_on_inner == false;
   CHECK(factor_on_inner) << "only factor_on_inner = True is supported for now";
 
-  int size = this->stop() - this->start();
-  int split_count = size / factor;
-  int tail_size = size % factor;
-  int output_group_count = (tail_size > 0) ? 2 : 1;
+  auto size = this->stop() - this->start();
```

Comment:
Minor nit: within reason, I'd try to avoid "auto", unless it is an iterator. "auto" is easier for writer of the code, while explicit type is easier for readers of the code. Since the code is written once, and read many times, explicit types are slightly better.

Reply:
So, I kind of disagree with you about `auto` (I'm kind of in the almost-always-auto camp: https://herbsutter.com/2013/08/12/gotw-94-solution-aaa-style-almost-always-auto/).  Can we leave this in "reasonable people can disagree" territory? :-p